view/rooms/app.js

```diff
@@ -59,6 +59,11 @@ const BOT_REQ_HEADERS = {
   Expires: '-1',
   'Cache-Control': 'private, max-age=31536000', // 31536000s = 1 year
 };
+
+const openInNewTabProps = 'rel="noopener noreferrer" target="_blank"';
+let _refocusRoomsChatterUrl;
+let _refocusRoomsContactEmail;
+let banner;
```

Comment:
couldnt you just set as null here and then avoid the else line 1052

Reply:
+1